moovingon_ai/README.md

```diff
@@ -0,0 +1,47 @@
+# moovingon.ai
+
+## Overview
+[moovingon.ai][1] is a platform for cloud operations and NOC management. It consolidates alerts across your observability suite and associates them with automated runbooks for alert and incident remediation. By integrating this, you can utilize the power of Datadog and moovingon.ai for efficient, automated incident management.
+
+moovingon.ai uses Datadog's monitors, logs, and event data for alert correlation and aggregation.
```

Comment:
Can we change this to something like the below : 

This integration sends in moovingon.ai log and event data to Datadog such as " replace with types of data monitored". With this integration, you can utilize the out-of-the-box Datadog dashboard for alert correlation and aggregation.

Reply:
Not really, because it's pretty much what I wrote in line 10. 

"2. **Comprehensive incident management**: Log all remediation actions performed in moovingon.ai in Datadog events for compliance and remediation clarity."

These events are visible from the actual monitors that were triggered in Datadog or inside the moovingon.ai Overview dashboard.

Line 6 is referring to moovingon.ai itself so if I change it it's like I'm write the same thing twice. you can suggest a change for line 10 if you think it should change.

Reply:
Thanks for clarifying! If the integration doesn't include an ootb monitor, we should be clear that they can create Datadog monitors to alert or remove monitors from the sentence

Reply:
@emarsha94 it's optional. The user can either mark all the monitors or select whatever monitors he wants. I wouldn't call it ootb because the user himself should decide if that's the behavior he wants or not. If you look at line 21, I already wrote about that:
4. **(Optional)** To receive events from all monitors, click on **install/update the webhook**.

Reply:
I rewritten step 4 to be more precise about what the user have to do.